### ISSUE TITLE
[Enhance] Toggle's link will be a <button> - React

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2945]
**User Story:**
As Spark we want Toggle to use a button to open and close so that we are accessible. 

**Notes:**
Current State: Toggle uses a link to open and close it, since it doesn't navigate, it should be a button. 
Requestor: A11y audit, Section RM-12.
SME: Bob

**AC:**
React Toggle
- will see `<a>` replaced with `<button>` and fix the CSS.
- will see aria-controls. This can be added to the component with a lodash call if the ID doesn't exist.

aria-controls this attribute is on the trigger element (the button). It's value is set to the id of the content element. If the user provides an id use that. If the user doesn't provide an id, generate one with lodash.

```<button aria-controls="123"></button>
<div class="im_the_content" id="123">The content</div>
